echopype/convert/set_groups_azfp.py

```diff
@@ -512,18 +500,25 @@ def set_vendor(self) -> xr.Dataset:
         unpacked_data = self.parser_obj.unpacked_data
         parameters = self.parser_obj.parameters
         ping_time = self.parser_obj.ping_time
+        PHASE_PARAMS = ["burst_interval", "pings_per_burst", "average_burst_pings"]
+        PHASE_FREQ_PARAMS = [
+            "dig_rate",
+            "range_samples",
+            "range_averaging_samples",
+            "lock_out_index",
+            "gain",
+            "storage_format",
+        ]
         tdn = []
         for num in parameters["phase_number"]:
-            tdn.append(parameters["pulse_len_phase{}".format(num)][self.freq_ind_sorted] / 1e6)
+            tdn.append(parameters[f"pulse_len_phase{num}"][self.freq_ind_sorted] / 1e6)
         tdn = np.array(tdn)
         for param in PHASE_FREQ_PARAMS:
             for num in parameters["phase_number"]:
-                parameters[param].append(
-                    parameters[param + "_phase{}".format(num)][self.freq_ind_sorted]
-                )
+                parameters[param].append(parameters[param + f"_phase{num}"][self.freq_ind_sorted])
         for param in PHASE_PARAMS:
             for num in parameters["phase_number"]:
-                parameters[param].append(parameters[param + "_phase{}".format(num)])
+                parameters[param].append(parameters[param + f"_phase{num}"])
```

Comment:
Use f strings for entire string you're composing, not just part of it:
```suggestion
                parameters[param].append(parameters[f"{param}_phase{num}"])
```

Reply:
Updated!